woocommerce/google-listings-and-ads
# Update apiclient library to version 2.10

### Changes proposed in this Pull Request:

This PR updates the Google apiclient library to version 2.10

```
composer update google/apiclient --with-dependencies

- Upgrading firebase/php-jwt (v5.2.1 => v5.4.0)
- Upgrading google/apiclient (v2.9.1 => v2.10.0)
- Upgrading google/apiclient-services (v0.164.0 => v0.200.0)
- Upgrading google/auth (v1.15.0 => v1.16.0)
- Upgrading guzzlehttp/guzzle (7.2.0 => 7.3.0)
- Upgrading guzzlehttp/psr7 (1.7.0 => 1.8.2)
- Upgrading psr/log (1.1.3 => 1.1.4)
```

There are a few minor changes from Guzzle 7.2 > 7.3, since we replace the namespace in this library it needed a few minor adjustments in the script we use to replace the namespace.

The biggest change is that the apiclient services change to using namespaces, which is what we take advantage of in this PR.

Closes #814

### Detailed test instructions:

1. Checkout this PR and run `composer install`
2. Test all requests to the merchant center services
3. Onboarding of an existing MC account (goes through all the site verification steps)
4. Syncing products
5. Clear the product status transient on the connection test page and reload the Product Feed page (goes through all the product status requests)
6. Confirm that we don't receive any warning/errors



### Changelog entry
* Update - Version 2.10 of the apiclient library.
* Update - Switch apiclient services to namespaces.
